internal/resources/payments/deployments.go

```diff
@@ -28,6 +34,58 @@ func getEncryptionKey(ctx core.Context, payments *v1beta1.Payments) (string, err
 	return "", nil
 }
 
+func temporalEnvVars(ctx core.Context, stack *v1beta1.Stack, payments *v1beta1.Payments) ([]v1.EnvVar, error) {
+
+	temporalURI, err := settings.RequireURL(ctx, stack.Name, "temporal", "dsn")
+	if err != nil {
+		return nil, err
+	}
+
+	if err := validateTemporalURI(temporalURI); err != nil {
+		return nil, err
+	}
+
+	if secret := temporalURI.Query().Get("secret"); secret != "" {
+		_, err = resourcereferences.Create(ctx, payments, "payments-temporal", secret, &v1.Secret{})
+	} else {
+		err = resourcereferences.Delete(ctx, payments, "payments-temporal")
+	}
+	if err != nil {
+		return nil, err
+	}
+
+	env := make([]v1.EnvVar, 0)
+	env = append(env,
+		core.Env("TEMPORAL_TASK_QUEUE", stack.Name),
```

Comment:
Are you sure it will not conflict with orchestration?

Reply:
I just removed it, we don't need it in payments